app/models/contribution.rb

```diff
@@ -21,6 +21,10 @@ class Contribution < ApplicationRecord
   belongs_to :repository
   belongs_to :reviewed_by, class_name: "User", foreign_key: "reviewer_id", optional: true
 
+  has_many :contributions_users, dependent: :destroy
+  has_many :users, through: :contributions_users
```

Comment:
I thought about that but i was getting confused on how i would associate all the authors to the contribution through their id. I personally never used a HABTM association before, i should search for some examples

Reply:
So searching for some examples and testing simple demos i decided to keep that way at least for now due to the amount of reworking that this would cause, plus for the feature tests on admin/contributions i would need to modify the `users` factory

Reply:
No problem! It was just a suggestion, the code is great as it is!

Reply:
Thanks to your suggestion i could finally understand how HABTM experimenting on rails console, thanks for your attention!

Reply:
HABTM would fit here